samples/Mvc/Mvc.Client/Properties/launchSettings.json

```diff
@@ -1,4 +1,12 @@
 {
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:7731/",
```

Comment:
This URL doesn't match the port we expect in the OIDC configuration. Please consider fixing it.